tests/foreman/api/test_config_template.py

```diff
@@ -30,3 +30,57 @@ def test_build_pxe_default(self):
         )
         response.raise_for_status()
         self.assertIsInstance(response.json(), dict)
+
+    def test_associate_config_template_with_organization(self):
+        """@Test: Associate config template with organization
+
+        @Assert: Config template is associated with organization
+
+        @Feature: ConfigTemplate
+
+        """
+        org_1_id = entities.Organization().create_json()['id']
+        org_2_id = entities.Organization().create_json()['id']
+
+        # By default, a configuration template should have no organizations.
+        conf_templ = entities.ConfigTemplate().create_json()
+        self.assertEqual(0, len(conf_templ['organizations']))
+
+        # Associate our configuration template with one organization.
+        client.put(
+            entities.ConfigTemplate(id=conf_templ['id']).path(),
```

Comment:
As of [NailGun 0.7.0](https://github.com/SatelliteQE/nailgun/releases/tag/0.7.0), you can now write this instead:

    content_view = entities.ContentView(organization=self.org.id).create()

What's the practical upshot? You can write code like this:

```python
conf_templ = entities.ConfigTemplate().create()
self.assertEqual(0, len(conf_templ.organizations))
client.put(
    conf_templ.path()
    ...
)
```

Here's a capture from my local system that further illustrates what's going on:

```python
>>> from robottelo import entities
>>> content_view = entities.ContentView(organization=1).create()
>>> type(content_view) == entities.ContentView
True
>>> content_view.id
1043
>>> content_view.organization.id
1
>>> content_view.get_values()
{'description': None, 'repository': [], 'composite': None, 'component': [], 'label': u'bed9f251-953f-4670-b973-150536e241a3', 'organization': <robottelo.entities.Organization object at 0x7f51ff0ffe90>, 'id': 1043, 'name': u'\u4718\U000269f5'}
```